sale_layout_hidden/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale Layout Hidden
+====================
+
+This module allows to Hide some sections in the printout of sale orders
+
```

Comment:
Can you add a description of why this is useful? Suggestion:

"""
This can be used to add service products in a quotation with a price of 0 which will create a task in a project without showing them to the customer. This is very useful when used in conjunction with product sets (see `sale_product_set_layout`).

The module also hides the lines from the invoice. 
"""